pages/api-reference/connect/webhook-subscriptions/index.mdx

```diff
@@ -43,6 +43,35 @@ Webhooks are delivered in near real-time as events tied to your application are
+
+#### Funding Sources
+
+| Event Topic Name                      | Description                                                                                                                                                                                                                                                                                                                                                                                                            |
+| ------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| funding_source:created                | **Description:** A funding source was added to a main Dwolla Account.<br /> **Timing:** Occurs upon a POST request to the [Create a funding source for an account](https://developers.dwolla.com/api-reference/connect/funding-sources/https://developers.dwolla.com/api-reference/connect/funding-sources/create-for-account) endpoint, or when a funding source is added via a third party bank verification method. |
+| funding_source:removed                | **Description:** A funding source was removed from a main Dwolla Account.<br /> **Timing:** Occurs upon a POST request to the [Remove a funding source](https://developers.dwolla.com/api-reference/connect/funding-sources/remove) endpoint.                                                                                                                                                                          |
+| external_party.funding_source:created | **Description:** A funding source was added to an External Party.<br /> **Timing:** Occurs upon a POST request to the [Create a funding source for an external party](https://developers.dwolla.com/api-reference/connect/funding-sources/create-for-external-party) endpoint, or when a funding source is added via a third party bank verification method.                                                           |
+| external_party.funding_source:removed | **Description:** A funding source was removed from a main Dwolla Account.<br /> **Timing:** Occurs upon a POST request to the [Remove a funding source](https://developers.dwolla.com/api-reference/connect/funding-sources/remove) endpoint.                                                                                                                                                                          |
```

Comment:
| external_party.funding_source:removed | **Description:** A funding source was removed from a main Dwolla Account.<br /> **Timing:** Occurs upon a POST request to the [Remove a funding source](https://developers.dwolla.com/api-reference/connect/funding-sources/remove) endpoint. |
```suggestion
| external_party.funding_source:removed | **Description:** A funding source was removed from a main Dwolla Account.<br /> **Timing:** Occurs upon a POST request to the [Remove a funding source](/api-reference/connect/funding-sources/remove) endpoint.                                                                                                                                                                          |
```